src/pal/inc/pal.h

```diff
@@ -2736,11 +2739,10 @@ typedef struct DECLSPEC_ALIGN(16) _CONTEXT {
     };
 
     //
-    // Vector registers.
+    // Upper half of YMM registers.
     //
 
-    M128A VectorRegister[26];
-    DWORD64 VectorControl;
+    M128A Ymmh[16];
```

Comment:
Actually, maybe it would be better to preserve the original way - the VectorRegister and VectorControl to keep it on par with Windows.